tests/integration/profiling/rule_based_profilers/test_profiler_user_workflows.py

```diff
@@ -134,7 +134,13 @@ def test_alice_profiler_user_workflow_single_batch(
         data_context=data_context,
     )
 
-    profiler.run()
+    # BatchRequest yielding exactly one batch
+    alice_single_batch_data_batch_request: dict = {
+        "datasource_name": "alice_columnar_table_single_batch_datasource",
+        "data_connector_name": "alice_columnar_table_single_batch_data_connector",
+        "data_asset_name": "alice_columnar_table_single_batch_data_asset",
+    }
+    profiler.run(batch_request=alice_single_batch_data_batch_request)
```

Comment:
I must say, I really do like this flow a lot better. Thank you @alexsherstinsky

Reply:
🙇